src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java

```diff
@@ -1065,9 +1065,9 @@ private static String forAgency(final String agencyId) {
 
     public InmateDetail getOffender(final String offenderNo) {
         telemetryClient.trackEvent("getOffender 1.1_beta version used");
-        return offenderRepository.findRootOffenderByNomsId(offenderNo)
+        return offenderBookingRepository.findLatestOffenderBookingByNomsId(offenderNo)
                 .map(offenderTransformer::transform)
-                .orElseThrow(EntityNotFoundException.withId(offenderNo));
+                .orElseGet(() -> offenderTransformer.transformWithoutBooking(offenderRepository.findRootOffenderByNomsId(offenderNo).orElseThrow()));
```

Comment:
Is this a breaking change? Although I guess if you think it's not used then no matter.